cloud/app/server/docker.js

```diff
@@ -279,4 +281,11 @@ const allocatePorts = async (count = 1) => {
   return rtv;
 };
 
-module.exports = { ensureRunning, stop, RUN_DIR };
+/** list all capability containers */
+const listCapabilityContainers = async () => {
+  const list = await docker.listContainers({all: true});
```

Comment:
why `all: true`, this will list stopped containers as well (https://docs.docker.com/reference/api/engine/version/v1.37/#tag/Container/operation/ContainerList).